### PR TITLE
fix to ins info, add city for display

### PR DESCRIPTION
**Pull request by stephenwaite**

### CODE REVIEW

File: library/patient.inc [hunk 1]
```diff
@@ -110,7 +110,8 @@ function getInsuranceProvidersExtra() {
 // add a global and if for where to allow inactive inscompanies
 
         $sql = "select insurance_companies.name, insurance_companies.id, " .
-          "addresses.zip, addresses.line1, addresses.state " .
+          "addresses.line1, addresses.line2, addresses.city, " .
```
Comment (robertdown) [inline library/patient.inc]: You query for `addresses.line2` here but don't actually return it on line 141.

File: library/patient.inc [hunk 2]
```diff
@@ -137,6 +138,10 @@ function getInsuranceProvidersExtra() {
            "," . $row['zip'] . ")";
         break;
         case $GLOBALS['insurance_information'] = '5':
+	       $returnval[$row['id']] = $row['name'] . " (" . $row['line1'] . "," . $row['city'] .
+               "," . $row['state'] . "," . $row['zip'] . ")";
```
Comment (robertdown) [inline library/patient.inc]: No space between comma and next word?

This is returning `Name (Line1,City,ST,ZIP)` which looks cramped, suggest `Name (Line1, City, ST ZIP)

**Comment by robertdown** [comment]
I went ahead and just made the edit since it was so simple. But I'd like to try something, @stephenwaite if you look on the right sidebar here on Github, the last thing (under the participants list should be a checkbox to "Allow edits from maintainers" -- can you check that and let me see if I can push my edit to *your* branch which in theory would update this PR (Could prove a very beneficial tool for future code reviews)

Here is a screenshot from one of my PRs as a reference 
screen shot 2017-04-01 at 02 07 30

**Comment by stephenwaite** [comment]
sure, it's checked, thanks for your suggestions and edits

**Comment by robertdown** [comment]
Merged - thanks for the contribution @stephenwaite

**Pull request merged**